FAQ.md

```diff
@@ -0,0 +1,23 @@
+## FAQ
```

Comment:
Adding a FAQ on the main readme was a specific request of Brian's - so I'd like for there to at least be a reference to this file. I also think the FAQ is really helpful with honing the pitch for MobileCoin so the reader is armed with conversational-tone answers to common questions.